site/content/how-to/traffic-management/advanced-routing.md

```diff
@@ -1,20 +1,23 @@
 ---
-title: "Routing to Applications Using HTTP Matching Conditions"
-description: "Learn how to deploy multiple applications and HTTPRoutes with request conditions such as paths, methods, headers, and query parameters"
+title: "Application routes using HTTP matching conditions"
 weight: 200
 toc: true
 docs: "DOCS-1422"
 ---
 
+Learn how to deploy multiple applications and HTTPRoutes with request conditions such as paths, methods, headers, and query parameters
+
+## Overview
+
 In this guide we will configure advanced routing rules for multiple applications. These rules will showcase request matching by path, headers, query parameters, and method. For an introduction to exposing your application, we recommend that you follow the [basic guide]({{< relref "/how-to/traffic-management/routing-traffic-to-your-app.md" >}}) first.
 
 The following image shows the traffic flow that we will be creating with these rules.
 
-{{<img src="img/advanced-routing.png" alt="Traffic Flow Diagram">}}
+{{<img src="img/advanced-routing.png" alt="">}}
 
 The goal is to create a set of rules that will result in client requests being sent to specific backends based on the request attributes. In this diagram, we have two versions of the `coffee` service. Traffic for v1 needs to be directed to the old application, while traffic for v2 needs to be directed towards the new application. We also have two `tea` services, one that handles GET operations and one that handles POST operations. Both the `tea` and `coffee` applications share the same Gateway.
 
-## Prerequisites
+## Before you begin
```

Comment:
Personally, I like the idea of "Before you begin" > "Prerequisites", but I do not see it in any of our style guides.

(OTOH, readers of software docs may be "used to" seeing "Prerequisites")

Reply:
It's part of the templates we've been standardising the documentation towards.

There are multiple other things that need to be changed, but this PR is intended to stay closely tight to the scope of the original filed issue.